packages/shared/src/components/modals/FeedFiltersModal/FeedFiltersIntroModal.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactElement, useContext } from 'react';
+import classNames from 'classnames';
+import { ModalProps } from '../StyledModal';
+import { ResponsiveModal } from '../ResponsiveModal';
+import UserIcon from '../../../../icons/user.svg';
+import { Button } from '../../buttons/Button';
+import FeedFiltersIntroModalTags from './FeedFiltersIntroModalTags';
+import { FeedFiltersIntroModalTagsContainer } from '../../utilities';
+import { Features, getFeatureValue } from '../../../lib/featureManagement';
+import FeaturesContext from '../../../contexts/FeaturesContext';
+
+type FeedFiltersProps = {
+  actionToOpenFeedFilters: () => unknown;
+  feedFilterModalType: string;
+};
+
+type ModalFooterProps = {
+  feedFilterOnboardingModalType: string;
+  onOpenFeedFilterModal: (event: React.MouseEvent<Element, MouseEvent>) => void;
+} & Pick<ModalProps, 'onRequestClose'>;
+
+type FeedFiltersIntroModalProps = FeedFiltersProps &
+  ModalFooterProps &
+  ModalProps;
+
+const footerClass = {
+  introTest1: 'justify-center',
+  introTest2: 'justify-between',
+};
+
+const IntroModalFooter = ({
+  feedFilterOnboardingModalType,
+  onRequestClose,
+  onOpenFeedFilterModal,
+}: ModalFooterProps) => {
+  return (
+    <footer
+      className={classNames(
+        footerClass[feedFilterOnboardingModalType],
+        'flex fixed responsiveModalBreakpoint:sticky bottom-0 py-3 border-t border-theme-divider-tertiary bg-theme-bg-tertiary',
+      )}
+    >
+      {feedFilterOnboardingModalType === 'introTest2' && (
+        <Button
+          className="ml-4 w-20 text-theme-label-tertiary btn-default"
+          onClick={onRequestClose}
+        >
+          Skip
+        </Button>
+      )}
+
+      <Button
+        className="mr-4 w-40 btn-primary-cabbage"
+        onClick={onOpenFeedFilterModal}
+      >
+        {feedFilterOnboardingModalType === 'introTest1'
+          ? 'Create my feed'
```

Comment:
This probably also needs the MyFeedButtonCopy ab test

Reply:
Even if it is the button that only opens the actual Feed filter?